src/styles/components/edit-work-order-description.scss

```diff
@@ -0,0 +1,10 @@
+.button-group {
+    display: flex;
+    gap: 1rem; 
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  .primary-button {
+    margin: 0;
+  }
```

Comment:
The class names should be more specific to the component (file name) youve used. Such as edit-description-button-group-container. But personally, I would set these styles inline. Saves you the faff